packages/renderer/src/lib/task-manager/TaskManagerItem.svelte

```diff
@@ -52,7 +57,9 @@ async function doExecuteAction(task: TaskInfo) {
 <div class="flex flew-row w-full py-2">
   <!-- first column is the icon-->
   <div class="flex w-3 {iconColor} justify-center">
+    <div class="align-top" role="img" aria-label="Icon status of task {task.id}">
```

Comment:
does not make much sense to have the task id in the aria label? ids are internal, task name would make more sense about accessibility

Reply:
task name then ?

Reply:
I was wondering if I should display the status as part of the label because it won't help much to know there is an icon

Reply:
Maybe yes, or just `task icon status`, I am also not sure we need the `role=img` as the <Fa> component already has it (from my memory, need to be fact checked)

Reply:
updated